src/gui/tray/UserModel.cpp

```diff
@@ -56,7 +56,7 @@ User::User(AccountStatePtr &account, const bool &isCurrent, QObject *parent)
 
     connect(_account->account().data(), &Account::accountChangedAvatar, this, &User::avatarChanged);
     connect(_account.data(), &AccountState::statusChanged, this, &User::statusChanged);
-
+    
```

Comment:
This change shouldn't be here, something decided to introduce trailing spaces behind your back. ;-)